xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -105,6 +113,23 @@
   private final Map<URI, CacheResourceConfigurationParser> resourceXmlParsers = new HashMap<URI, CacheResourceConfigurationParser>();
   private final ConfigType config;
 
+  static String replaceProperties(String originalValue, final Properties properties) {
```

Comment:
Well, grows what the test has to assert about. This only tests the resolving logic.